ui/src/app/pages/signed-in/nav-bar.tsx

```diff
@@ -239,7 +241,7 @@ export const NavBar = withUserProfile()(
         {
           this.state.sideNavVisible
           && <SideNav
-            profile={this.props.profile}
+            profile={this.props.profileState.profile}
```

Comment:
bug in nav menu - it was not looking in the right place for the profile store, so the menu would never update on a state change